runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/StageInfo.java

```diff
@@ -117,6 +119,24 @@ public List<PerStepNamespaceMetrics> extractPerWorkerMetricValues() {
     Iterables.addAll(
         metrics,
         StreamingStepMetricsContainer.extractPerWorkerMetricUpdates(metricsContainerRegistry()));
+    translateKnownPerWorkerCounters(metrics);
     return metrics;
   }
+
+  private void translateKnownPerWorkerCounters(List<PerStepNamespaceMetrics> metrics) {
+    for (PerStepNamespaceMetrics perStepnamespaceMetrics : metrics) {
+      if (!BigQuerySinkMetrics.METRICS_NAMESPACE.equals(
+          perStepnamespaceMetrics.getMetricsNamespace())) {
+        continue;
+      }
+      for (MetricValue metric : perStepnamespaceMetrics.getMetricValues()) {
+        if (BigQuerySinkMetrics.THROTTLED_TIME.equals(metric.getMetric())) {
+          Long msecs = metric.getValueInt64();
```

Comment:
nit: use primitive `long`

Reply:
Leaving this as is.
`getValueInt64` returns a Long, but it's possible - though unexpected - for the function to return null.

Added a test case to cover this case.